server/athenian/api/controllers/dashboard_controller.py

```diff
@@ -66,7 +65,8 @@ async def create_dashboard_chart(
     dashboard = await _get_request_dashboard(request, team_id, dashboard_id)
```

Comment:
for example, we should load the prefixer at this line, in the very first async call